resources/sass/app.scss

```diff
@@ -1,10 +1,22 @@
+@tailwind base;
+
+@tailwind components;
+
+@tailwind utilities;
+
+// Old stuff. All future dev in Tailwind, please.
+
 // Bootstrap & Flatly ( http://bootswatch.com/flatly/ )
 @import "variables", "bootstrap", "bootswatch";
 
 $link_color: #428bca;
 $font-family-oswald: 'Oswald', 'Trebuchet MS', 'Open Sans', arial, sans-serif;
 
 /* Build our stylesheet, which currently merely extends bootstrap styles. */
+html {
+    font-size: 16px;
```

Comment:
Tailwind assumes this base font size but for some reason the base we had been using prior (a third party base) set it to 10px. ¯\(°_o)/¯